plugins/misc/telegraph/__main__.py

```diff
@@ -69,7 +71,25 @@ async def telegraph_(message: Message):
             else:
                 text = content
                 header = "Pasted content by @theuserge"
-        t_url = await pool.run_in_thread(post_to_telegraph)(header, text.replace("\n", "<br>"))
+        at = None
+        if TE_LEGRA_PH_ACCESS_TOKEN != 0:
+            at = (
+                await message.client.get_messages(
+                    config.LOG_CHANNEL_ID,
+                    TE_LEGRA_PH_ACCESS_TOKEN
+                )
+            ).text
+        else:
+            at = await pool.run_in_thread(create_access_token)(header)
+            await message.client.send_message(
+                config.LOG_CHANNEL_ID,
+                at
+            )
```

Comment:
This seems spammy cuz it will send a token on each restart of server

Reply:
after receiving once, we can update env var 😅

Reply:
i guess better solution would be someone experienced with mongoDB could PR .